allennlp/tests/models/simple_tagger_test.py

```diff
@@ -71,6 +71,20 @@ def test_regularization(self):
         numpy.testing.assert_almost_equal(training_loss, validation_loss)
 
 
+class SimpleTaggerSpanF1Test(ModelTestCase):
+    def setUp(self):
+        super(SimpleTaggerSpanF1Test, self).setUp()
```

Comment:
@nelson-liu, quick tip, you can just say `super().setUp()` here and get the same effect.